src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/SQLFunctions.java

```diff
@@ -1023,4 +1015,31 @@ public String getCastScriptStatement(String name, String castType, List<KVValue>
                 );
         }
     }
+
+    public static Schema.Type getOrderByFieldType(Field field) {
```

Comment:
Could you add JavaDoc here to clarify that why we stick with hardcoding return type here? Basically I think it's because there are only 2 order by type TEXT and NUMBER in ES so what is returned here is essentially the category of function rather than the actual return type.

Reply:
good idea- will do